--- uio	2020-07-30 02:20:04.019981652 +0300
+++ uiautomatorviewer	2020-07-30 02:20:11.496999204 +0300
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
 #
 # Copyright 2012, The Android Open Source Project
 #
@@ -18,7 +18,7 @@
 # and set up progdir to be the fully-qualified pathname of its directory.
 prog="$0"
 while [ -h "${prog}" ]; do
-    newProg=`/bin/ls -ld "${prog}"`
+    newProg=`ls -ld "${prog}"`
     newProg=`expr "${newProg}" : ".* -> \(.*\)$"`
     if expr "x${newProg}" : 'x/' >/dev/null; then
         prog="${newProg}"
